src/whip-client.c

```diff
@@ -128,6 +128,7 @@ static GOptionEntry opt_entries[] = {
 	{ "turn-server", 'T', 0, G_OPTION_ARG_STRING_ARRAY, &turn_server, "TURN server to use, if any; can be called multiple times (turn(s)://username:password@host:port?transport=[udp,tcp])", NULL },
 	{ "force-turn", 'F', 0, G_OPTION_ARG_NONE, &force_turn, "In case TURN servers are provided, force using a relay (default: false)", NULL },
 	{ "log-level", 'l', 0, G_OPTION_ARG_INT, &whip_log_level, "Logging level (0=disable logging, 7=maximum log level; default: 4)", NULL },
+	{ "eos-sink-name", 'E', 0, G_OPTION_ARG_STRING, &eos_sink_name, "GStreamer sink name for EOS signal", NULL },
```

Comment:
Why not a `-e` instead of `-E`, since the non-capitalized version is not used yet? Even though, now that I look at it, there's capitalized versions of some other properties too where there's no conflict, so feel free to disregard.

Reply:
changed to `-e`